RiotSharp/RiotApi.cs

```diff
@@ -42,12 +43,14 @@ public class RiotApi : IRiotApi
         private const string GameRootUrl = "/api/lol/{0}/v1.3/game";
         private const string RecentGamesUrl = "/by-summoner/{0}/recent";
 
-        private const string LeagueRootUrl = "/api/lol/{0}/v2.5/league";
-        private const string LeagueChallengerUrl = "/challenger";
-        private const string LeagueMasterUrl = "/master";
+        private const string LeagueRootUrl = "/lol/league/v3";
+        private const string LeagueChallengerUrl = "/challengerleagues/by-queue/{0}";
+        private const string LeagueMasterUrl = "/masterleagues/by-queue/{0}";
+
+        private const string LeagueBySummonerUrl = "/leagues/by-summoner/{0}";
 
```

Comment:
I think we can remove those empty lines now

Reply:
Which ones ?

Reply:
This one and the one above `LeagueBySummonerUrl`

Reply:
So line 45 and 49 ?
I'm okay with the 49nth but I think we should keep the 45nth.

Reply:
I meant 49 and 51, so that all the league-related urls are grouped

Reply:
Ah yeah. My phone is not showing the right lines.